_i18n/de/resources/moneropedia/clearnet.md

```diff
@@ -1,32 +1,33 @@
+Wenn du das Internet für Dinge wie das Abrufen der Nachrichten, E-Mails, soziale Medien oder auch Monero nutzt, nutzt du sehr wahrscheinlich eine Clearnet-Verbindung. Dies bedeutet, dass deine *gesamten* Verbindungen aufgespürt, verfolgt und überwacht werden können, und das von:
 
-- your [ISP](https://en.wikipedia.org/wiki/ISP)
-- the website/service/person you're communicating with
-- possibly a [Five Eyes](https://en.wikipedia.org/wiki/5_Eyes) capable entity
+- deinem [Internetprovider](https://de.wikipedia.org/wiki/Internetdienstanbieter)
+- der Webseite/der Person/dem Service, mit der/dem du in Verbindung stehst
+- eventuell einer mit den Möglichkeiten der [Five Eyes](https://en.wikipedia.org/wiki/5_Eyes) befähigten Instanz
 
-and even if you use [HTTPS](https://en.wikipedia.org/wiki/HTTPS) or similar (which *encrypts* your transmission), your route is not hidden nor is it anonymous, thus; it is in the *clear*.
+Selbst wenn du [HTTPS](https://de.wikipedia.org/wiki/Hypertext_Transfer_Protocol_Secure) oder Ähnliches (das deine Datenübertragung *verschlüsselt*) nutzt, ist dein Verlauf weder verborgen noch anonym; es ist also sichtbar und demnach im *"Clear"*.
 
-### In-depth information
+### Ausführliche Informationen
 
-Since a traditional [VPN](https://en.wikipedia.org/wiki/VPN) cannot save you from clearnet (as you are still using *clearnet* (though you are more proxied than without a VPN)), you should use an *anonymous overlay network* to avoid using clearnet directly:
+Da dich ein klassisches [VPN](https://de.wikipedia.org/wiki/Virtual_Private_Network) nicht vor dem Clearnet schützen kann - du nutzt schließlich nach wie vor das *Clearnet* (wobei deine Daten zumindest besser umgeleitet werden, als ohne die Verwendung eines VPNs) - solltest du ein anonymes Overlay-Netzwerk nutzen, um den direkten Gebrauch des Clearnets zu vermeiden:
 
 - @Kovri
-- @Java-I2P
-- [Tor](https://torproject.org/)
+- Java-I2P
+- [Tor](https://torproject.org/de/)
 
-These technologies protect you from clearnet by building an anonymous network **over** clearnet to keep your transmissions both encrypted **and** anonymous.
+Diese Technologien schützen dich vor dem Clearnet, indem sie ein anonymes Netz **über** ebendiesem errichten und deine Verbindungen somit verschlüsselt **und** anonym bleiben.
 
-Here is an accurate, [interactive diagram](https://www.eff.org/pages/tor-and-https) provided by the [EFF](https://www.eff.org/) which describes *clearnet* as it relates to **Tor**. The concept also (somewhat) applies to @Kovri and @I2P in terms of anonymity with the exception that:
+Ein [interaktives Diagramm](https://www.eff.org/pages/tor-and-https) des [EFF](https://www.eff.org/) zeigt den Zusammenhang des *Clearnets* und **Tor**. Dieses Konzept trifft in puncto Anonymität (in gewisser Hinsicht) auch auf @Kovri und I2P zu, mit der Ausnahme, dass:
+
+- Kovri Nodes nicht beendet, wenn zu einer Eepsite verbunden wird
```

Comment:
```suggestion
- Kovri für Verbindungen zu @eepsite keine Exit-Knoten verwendet
```

I have no idea what an eepsite is?

Reply:
Eepsites are very basically websites on the I2P-network. I left out the @ because the Moneropedia entry concerning eepsites was removed, thus the reference doesn't lead anywhere or rather doesn't exist (same for I2P).